vassal-app/src/main/java/VASSAL/build/module/chessclockcontrol/ChessClock.java

```diff
@@ -996,13 +1037,12 @@ public void actionPerformed(ActionEvent e) {
 
       Command c = new NullCommand();
       final ChessClockControl ccc = ChessClockControl.getInstance();
-      if (command.contains(CHESSMENU_START) || command.contains(CHESSMENU_STOP)) {
-        if ((ccc != null) && ccc.getClocksTicking() > 0) {
-          c = c.append(ChessClockControl.getInstance().startNextClock());
-        }
-        else {
-          c = c.append(updateState(true));
-        }
+      if (ccc == null) return;
```

Comment:
The fact that `ccc.getClocksTicking()` is no longer checked seems to change the behavior in all cases rather than in some specific new case?

Reply:
It does - as written they seem like placeholders, as both start and stop commands are undifferentiated overloads of the same functionality available under hotkey control in ChessClockControl.  As written, they provide clearly differentiated functionality from what's available from the Control and more closely match the described functionality.  The intent is that the end state after a call to a ChessClock menu option function will be independent of other clock states, while ChessClockControl commands handle multi-clock and state dependent calls.

Reply:
That seems like it could be confusing to users who are expecting this to work "like a physical chess clock" where starting one clock necessarily stops the other and vice versa. I realize there may be other use cases where different functionality is desired (e.g. can force one clock to start/stop independently of whether another is), but it seems like that might be better addressed by creating additional menu options (e.g. "Start only this clock", or "Force-start only this clock" or something like that) and leaving the other ones working "like a physical chess clock".

Reply:
Interesting - the original impetus of this update was to make the clock work more like a physical chess clock.  Normally, if you keep hitting your clock button, it will never swap the clock back to your side - it always sends it over to your opponent.  Being able to just tap 'your' chess clock button doesn't require you to double check who the clock is now on - it is always going to be the opponent.  I suppose it ultimately doesn't matter what the options are labeled as long as they're all exposed to the user, so whatever is less confusing makes sense.

Reply:
I was mainly thinking about the ability to have more than one clock running at once (which doesn't happen with a physical clock). But likewise the ability to stop all the clocks -- which you can do -- nonetheless seems like a separate/different function.